pegasus/sites.v3/hourofcode.com/public/survey/splat.haml

```diff
@@ -151,33 +153,35 @@ rightbar: blank
           %option{value: label}= label
 
     %div#teacher-how-heard-other-wrapper{style: 'display: none;'}
-      %label.control-label{for: "teacher-how-heard-other"}<
+      %label.control-label{for: "teacher-how-heard-other"}
         %em{style: 'font-weight: normal;'} Other method
       %div
         %input#tacher-how-heard-other.form-control{name: "teacher_how_heard_other_s", placeholder: "", type: "text"}/
 
   .form-group
-    #teacher-district-wrapper{style: "display: none;"}
-      %label.control-label{for: "teacher-district"} What school district do you teach in? (optional)
-      %div
-        %input#teacher-district.form-control{name: "teacher_district_s", placeholder: "", type: "text"}/
+    %label.control-label{for: "hoc-centerpiece"}
+      Should we keep the Hour of Code as the centerpiece activity of Computer Science Education Week,
+      and keep “Hour of Code” as the unifying brand for all the other activities that educators organize throughout the week?
+    %span.form-required-field *
+    %div
+      %select#hoc-centerpiece.form-control{name: "hoc_centerpiece_s", type: "select", required: true}
+        %option{value: nil, selected: true}
+        - HocSurvey2015.hoc_centerpiece_values.each  do |label|
+          %option{value: label}= label
 
   .form-group
-    %label.control-label{for: "prize-choice"} What prize would you like to receive?
-    %span.form-required-field *
-    .radio-row
-      %label{class: "radio-inline"}
-        %input{type: "radio", id: "prize-choice-dropbox", name: "prize_choice_s", value: "Dropbox"} 10 GB of Dropbox space
-      %label{class: "radio-inline"}
-        %input{type: "radio", id: "prize-choice-skype", name: "prize_choice_s", value: "Skype"} $10 of Skype credit
-      %label{class: "radio-inline"}
-        %input{type: "radio", id: "prize-choice-none", name: "prize_choice_s", value: "None"} No prize
+    %label.control-label{for: "event-improvement"} How do you think we could improve the Hour of Code? (optional, limit 250 words)
+    %div
+      %textarea#event-improvement.form-control{name: "event_improvement_s", placeholder: "", style: "height: 8em;"}
 
   %input{name:'code_s', type:'hidden', value:code}
 
   .form-group.submit-btn
     %button#btn-submit.btn.btn-default{type: "submit"} Submit
 
+#prizeChoice{:style=>"display: none;"}
```

Comment:
is this used? I don't see a reference to prizeChoice in this commit, is it existing code?